### PR TITLE
examples/mediaplayer: Set source at PLAYER_START

### PR DESCRIPTION
**Pull request by junmin-kim**
To prepare mediaplayer, datasource should be set.

### CODE REVIEW

**Comment by seinfra** [comment]
Target : [575a6ad5ffee766dd5be0959b03ff77355d38b45] - Code Rule Check OK.

**Comment by seinfra** [comment]
Target : [575a6ad5ffee766dd5be0959b03ff77355d38b45] - Code Rule Check (C++) OK.

File: apps/examples/mediaplayer/mediaplayer_main.cpp [hunk 1]
```diff
@@ -130,6 +144,7 @@ void MyMediaPlayer::doCommand(int command)
 		if (mp.unprepare() != PLAYER_OK) {
 			cout << "Mediaplayer::unprepare failed" << endl;
 		}
+		mp.setDataSource(makeSource());
```
Comment (JeongJunSik) [inline apps/examples/mediaplayer/mediaplayer_main.cpp]: i think this logic should be called at start rather than STOP.

**Comment by seinfra** [comment]
Target : [e7e2872b85a249050c8f82da0a233cdd90426f76] -

**Comment by seinfra** [comment]
Target : [7b7fe479efbaced548a5d6875d3921fc1ffec7c2] - Code Rule Check (C++) OK.

**Comment by seinfra** [comment]
Target : [7b7fe479efbaced548a5d6875d3921fc1ffec7c2] - Code Rule Check OK.

**Comment by seinfra** [comment]
Target : [4386de88d1606ee1bd9209c0dc16ae2e7dbe3375] - Code Rule Check OK.

**Comment by seinfra** [comment]
Target : [4386de88d1606ee1bd9209c0dc16ae2e7dbe3375] - Code Rule Check (C++) OK.

**Pull request merged**